webpack/scenes/FlatpakRemotes/Details/FlatpakRemoteDetails.js

```diff
@@ -49,7 +71,40 @@ export default function FlatpakRemoteDetails() {
       </GridItem>
 
       <GridItem span={12}>
-        <Title headingLevel="h1" size="2xl" ouiaId="flatpak-remote-title">{name}</Title>
+        <Flex>
+          <FlexItem>
+            <Title headingLevel="h1" size="2xl" ouiaId="flatpak-remote-title">{name}</Title>
+          </FlexItem>
+          <FlexItem align={{ default: 'alignRight' }}>
+            <Button
+              ouiaId="fr-details-scan-button"
+              style={{ marginLeft: 'auto' }}
+              onClick={() => {
+                setIsScanning(true);
+                dispatch(scanFlatpakRemote(
+                  frId,
+                  () => { setIsScanning(false); },
+                  () => setIsScanning(false),
+                ));
+              }
+            }
+              variant="primary"
+              aria-label="scan_flatpak_remote"
+              isLoading={isScanning}
```

Comment:
Can we disable the scan button while scanning? This would help prevent users from clicking it multiple times.
```suggestion
              isLoading={isScanning}
              isDisabled={isScanning}
```